dev/serve.vue

```diff
@@ -20,7 +20,7 @@ export default Vue.extend({
   },
   data() {
     return {
-      hereApiToken: process.env.VUE_APP_HEREAPI,
+      cesiumAccessToken: process.env.VUE_APP_CESIUM_ACCESS_TOKEN,
     }
```

Comment:
I can't say I fully understand this but I think there is enough there I could google to understand. Good clear code is what I'm trying to say!

Reply:
Thanks, that's good to hear. I find Vue is quite easy to read but its good to hear from someone that hasn't used it before